specs/commandline/sign.md

```diff
@@ -37,8 +38,52 @@ Flags:
       --plugin-config strings    {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
       --signature-format string  signature envelope format, options: 'jws', 'cose' (default "jws")
   -u, --username string          username for registry operations (default to $NOTATION_USERNAME if not specified)
+      --use-image-manifest       optional for using OCI image manifest to store signatures in a registry
 ```
 
+## Use OCI image manifest to store signatures
+
+By default, Notation uses [OCI artifact manifest][oci-image-manifest] to store signatures in the registries. For backwards compatibility, Notation supports using `OCI image manifest` to store the signature in the registries that implement partial or older versions of the OCI Image specification. Use flag `--use-image-manifest` to force Notation store the signatures using OCI image manifest. Note that Notation doesn't automatically switch to `OCI image manifest` for a failure of putting `OCI artifact manifest`.
+
+Registries MAY not implement or enable the `Referrers API`, which is used by the client to fetch the referrers. In the context of Notation, the referrers are signatures. Notation follows the fallback procedure defined in [OCI distribution spec][oci-backward-compatibility] if `Referrers API` is unavailable.
+
+### Set config property for OCI image manifest
+
+OCI image manifest requires additional property `config` of type `descriptor`, which is not required by OCI artifact manifest. Notation creates a default config descriptor for the user if the flag `--use-image-manifest` is used. User can specify custom configurations by using flag `--config-file` for specific needs, which overrides the default configurations.
```

Comment:
> User can specify custom configurations by using flag `--config-file` for specific needs, which overrides the default configurations.

Same as above. What's the purpose of customizing the config blob?

Reply:
Remove it as no user requirement.